publications-2020.bib

```diff
@@ -217,6 +217,23 @@ @Article{2020:benbow.kawama.ea:coupled
   publisher = {{MDPI} {AG}}
 }
 
+@InProceedings{2020:beuchler.kinnewig.ea:parallel,
+  author  = {Sven Beuchler and Sebastian Kinnewig and Thomas Wick},
+  title   = {Parallel domain decomposition solvers for the time harmonic Maxwell equations},
+  editor  = {Susanne C. Brenner and Eric Tsz Shun Chung and Axel Klawonn and Felix Kwok and Jinchao Xu and Jun Zou},
+  booktitle = {Domain Decomposition Methods in Science and Engineering XXVI},
+  year    = 2020,
+  series  = {Lecture Notes in Computational Science and Engineering},
+  volume  = 145,
+  pages   = {615--622},
+  address = {Hong Kong, China},
+  month   = dec,
+  publisher = {Springer International Publishing},
+  note    = {In press},
```

Comment:
Since it is in a published volume, with page numbers, it cannot be "in press" -- it has already been printed. "In press" means it is accepted and has gone through the editing process, but has not been published in print. It is today equivalent to "published online" with no volume/issue/pages associated yet. Just remove the note.

Same below.

Reply:
The volume is not published yet. In fact, the Springer website shows that its copyright year is 2023. I don't see a way to order a hardcopy of the proceedings yet:
https://link.springer.com/book/9783030950248
http://www.ddm.org/DD26/Proc-26.php

Hence I thought `in press` was fitting here. We will need to go over these entries again next year once the volume is actually available.